.github/workflows/tag.yml

```diff
@@ -0,0 +1,35 @@
+name: Tag
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-ecosystem/action-get-merged-pull-request@v1
+        id: get-merged-pull-request
+        with:
+          github_token: ${{ secrets.GH_PAT }}
+
+      - uses: actions-ecosystem/action-release-label@v1
+        id: release-label
+        if: ${{ steps.get-merged-pull-request.outputs.title != null }}
+        with:
+          github_token: ${{ secrets.GH_PAT }}
+          labels: ${{ steps.get-merged-pull-request.outputs.labels }}
+      
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      
+      - run: echo 'Dry Run PATCH' # cargo release patch --execute
+        if: ${{ steps.release-label.outputs.level == release/patch }}
+      - run: echo 'Dry Run MINOR' # cargo release minor --execute
+        if: ${{ steps.release-label.outputs.level == release/minor }}
+      - run: echo 'Dry Run MAJOR' # cargo release major --execute
+        if: ${{ steps.release-label.outputs.level == release/major }}
```

Comment:
If you just leave out `--execute` it will do a dry run.

Also, does this trigger the release action? My understanding is that actions cannot trigger other actions.